kolibri/core/assets/src/vue/icon-button.vue

```diff
@@ -28,8 +28,7 @@
     },
     computed: {
       hasIcon() {
-        // something of a hack but seems to work fine
-        return this.$el.querySelector('svg');
+        return this.$slots.default[0].tag === 'svg';
```

Comment:
I'm not familiar with the API, but does this definitely work when no icon is passed in? (I might expect `return this.$slots.default[0]` to be `undefined`?)

Also, maybe we should just check for the existence of something passed into the slot, rather than the specific tag name? e.g. it might take an `img` also?